### ISSUE TITLE
Boost URL with keywords

### ISSUE THREAD

**Issue by Marx1st** [yacy/yacy_search_server#168]
Ergänzung, das Forum funktioniert leider überhaupt nicht (SQL Fehler beim Absenden eines Beitrages... Darum schreibe ich hier, man möge mir verzeihen.)

Die Forumssuche funktioniert leider nicht ( ;) ), darum hoffe ich, dass das Thema nicht schon einmal behandelt wurde.

Ein Feature, das ich bisher nicht finden konnte, ist die Möglichkeit bestimmte URLs aufgrund von Suchbegriffen zu "boosten". Soll heißen, z.B. es sucht jemand auf unserer Firmenseite nach "jobs". Nun gibt es eine ganze Reihe von Artikeln über "Jobs" auf verschiedenen Seiten, die nun alle im Suchergebnis angezeigt werden. Die Seite mit den aktuell offenen Stellen ist da bestimmt auch bei, aber leider erst z.B. an Rang 10.
Ich möchte aber sicherstellen, dass die "Job-Seite" immer ganz oben oder in einem extra Bereich über den Suchergebnissen angezeigt wird, damit Nutzer nicht lange suchen müssen. Praktisch möchte ich eine Liste mit Begriffen anlegen (z.B. "jobs", "stellen", "Bewerbung", etc.) und wenn einer (oder mehrere) dieser Begriffe in der Suchanfrage enthalten ist, soll die Seite "http://jobs.meinetollefirma.de" an erster Stelle stehen. Unabhängig von der sonstigen "Relevanz" des Treffers.
Natürlich müsste es die Möglichkeit geben mehrere URLs mit unterschiedlichen "Keywords" zu definieren.

Ist dies mit YaCy möglich?

Viele Grüße 
Tobias

**Comment by luccioman** [comment]
Hallo @Marx1st, 
sorry to answer in English, my German is too bad.
Did you have a look at the 'Ranking and Heuristics' page (/RankingSolr_p.html) ? Maybe the "Boost Query" field could help you?

**Comment by Marx1st** [comment]
Yep, I did. But I AFAIK there one only can define a single boost for a few keywords and all the pages which contain those keywords are boosted. This is not what I am trying to achieve. :)

I'll try to summarize my post in english :)

(forums throw a SQL error, thats why I post here)

I want to define keywords <-> URL combos. If someone searches for "foo", one specific page should always be on top (or in a special area on top) of the search results. And I want to do this with several keywords and URLs.

**Comment by okybaca** [comment]
Machine translation of the question:
One feature that I haven't been able to find yet is the ability to "boost" certain URLs based on search terms. This means, for example, someone is looking for “jobs” on our company website. Now there are a whole series of articles about “jobs” on various sites, all of which are now displayed in the search results. The page with the current vacancies is definitely there too, but unfortunately only in 10th place, for example.
But I want to make sure that the "job page" is always displayed at the top or in a special area above the search results so that users don't have to search for a long time. Practically, I would like to create a list of terms (e.g. "jobs", "positions", "application", etc.) and if one (or more) of these terms is included in the search query, the page should be "http://jobs. "meintollefirma.de" comes first. Regardless of the other “relevance” of the hit.
Of course there should be the possibility to define several URLs with different “keywords”.

Is this possible with YaCy?

Best regards
Tobias
